cpp/README.rst

```diff
@@ -23,7 +23,7 @@ Before using the Code Examples, first complete the installation and setup steps
 Developer Guide.
 
 The Getting Started section covers how to obtain and build the SDK, and how to build your own
-code utilizing the SDK with a sample �Hello World�-style application.  A similar procedure is 
+code utilizing the SDK with a sample �Hello World�-style application.  A similar procedure is 

```

Comment:
Got some character encoding going on here...

Reply:
What on earth

Reply:
UPDATE: I played around with this long enough (pushing from local terminal, IDE, and within GitHub) to suspect that the problem is whatever encoding Steve pushed up originally, not the RST file itself or the way it's being encoded in GitHub.

Reply:
Should we resolve this conversation for now then?